modules/scripts/config.py

```diff
@@ -28,7 +30,7 @@ def update_dict(base, head):
 
 def _get_default_config(username, reponame):
     conf = {'metadata': {'projectname': reponame,
-                         'version': 'development',
+                         'version': datetime.now().strftime('%Y-%m-%d %H:%M:%S'),
```

Comment:
![](https://user-images.githubusercontent.com/8245662/27137482-b65d9702-513b-11e7-9b23-1b56f9880729.png)

Set it in a format that would seem relevant here.